client/router_create.go

```diff
@@ -176,7 +176,7 @@ func (cli *VanClient) GetVanControllerSpec(options types.SiteConfigSpec, van *ty
 			}
 			annotations = map[string]string{"service.alpha.openshift.io/serving-cert-secret-name": "skupper-controller-certs"}
 		}
-	} else if !options.ClusterLocal {
+	} else if options.Ingress == types.IngressLoadBalancerString {
```

Comment:
Isn't it supposed to be `options.Ingress != types.IngressNoneString`?

Reply:
well "yes and no", see the first "if" is asking if it is a route, then I thought the code is more clear if this "else" asks specifically what it wants to know, since " if no route, and no clusterlocal" -> means it is loadbalancer. 
The resultant behaviour is the same if I ask for your suggestion, or this one implemented currently.